e2e/cypress/integration/enterprise/guest_accounts/guest_invitation_ui_spec.js

```diff
@@ -272,4 +277,44 @@ describe('Guest Account - Guest User Invitation Flow', () => {
         // * Verify the content and message in next screen
         verifyInvitationSuccess(email, 'An invitation email has been sent.');
     });
+
+    it('MM-18050 Verify when different feature settings are disabled', () => {
+        // # Disable Guest Account Feature
+        changeGuestFeatureSettings(false, true);
+
+        // # Login again after changing feature flag and reload current page
+        cy.apiLogin('sysadmin');
+        cy.visit(`/${testTeam.name}`);
+
+        // # Open Invite People
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+        cy.get('#invitePeople').should('be.visible').click();
+
+        // * Verify if Invite Members modal is displayed when guest account feature is disabled
+        cy.getByTestId('invitationModal').within(($el) => {
+            cy.wrap($el).find('h1').should('have.text', 'Invite Members');
+        });
+
+        // * Verify Share Link Header and helper text
+        cy.getByTestId('shareLink').should('be.visible').within(($el) => {
```

Comment:
`within` already puts you in the scope of the initial element, so there should be no need to `cy.wrap`.
```suggestion
        cy.getByTestId('shareLink').should('be.visible').within(() => {
            cy.get('h2 > span').should('have.text', 'Share This Link')
            // etc...
        })
```

Reply:
My Bad. Please ignore. I was trying to do a `cy.find` instead of `cy.get`. It worked now. Thanks.